openpype/hosts/maya/plugins/create/create_animation.py

```diff
@@ -85,3 +80,12 @@ def get_instance_attr_defs(self):
         """
 
         return defs
+
+    def apply_settings(self, project_settings, system_settings):
+        super(CreateAnimation, self).apply_settings(
+            project_settings, system_settings
+        )
+        # Hardcoding creator to be enabled due to existing settings would
+        # disable the creator causing the creator plugin to not be
+        # discoverable.
+        self.enabled = True
```

Comment:
Wouldn't be better to remove `enabled` from settings?

Reply:
The issue is with existing database settings, where the enabled setting would be False.

Reply:
But the `enabled` is still in settings schema https://github.com/ynput/OpenPype/pull/5297/files#diff-c6017fd21a1f9bf3c79271e6fe7f506dbaf5b5468724e9649e4ba15524146c86R558 .

Reply:
Good spot! Removed.